test/errors/valueAsTypeExpr-detailed.good

```diff
@@ -0,0 +1,11 @@
+─── error in /Users/daniel.fedorin/Documents/software/chapel/modules/internal/ChapelDomain.chpl:373 ───
+  Cannot resolve call: no matching candidates
+
+─── error in valueAsTypeExpr.chpl:2 [ValueUsedAsType] ───
+  Type expression produces a value of type 'int(64)' while a type was expected.
```

Comment:
I'd avoid "produces" here which seems to imply something active.  Something like "type constraint/specifier is a value rather than a type" is more what expect here (and this error seems very similar to the previous incompatibleKinds cases, so I'd try to unify terminology/wordings used with those as much as possible).